### ISSUE TITLE
Can not reach the accuracy of ResNet in the published paper.

### ISSUE THREAD

**Issue by souxun2015** [arashno/tensorflow_multigpu_imagenet#5]
I use your code to train ResNet34 on ImageNet, but the accuracy of single-crop is 0.655 which is much lower than 0.75 the accuracy of the published paper. 

batch_size = 256,
epoch_number = 120
num_gpus = 2
lr = tf.train.piecewise_constant(epoch_number, [60,90,110], [0.1,0.01,0.001,0.0001],name='LearningRate')
wd =  tf.train.piecewise_constant(epoch_number,[60],[0.0001,0.0001],name='WeightDecay')

And in the 'common.py':
def resnetBlock():
  ....
    with tf.variable_scope('B')
        x = spatialConvolution(x,3,1,filters_out,weight_initializer=conv_weight_initializer,wd=wd) #"the kernel size should be 3"

For the 'data_loader.py', I add the code to preprocess images for multi-scale training.

These are what I changed.

**Comment by arashno** [comment]
I have to check the kernel size, thank you for catching that.
Are you using the exact hyperparameters of the published paper (LR and WD, num of epochs, number of batches per epoch)?
In the 'data_loader.py' are you using data augmentation or you cut it out?

**Comment by souxun2015** [comment]
I follow the training setup of the published paper. I just add the data augmentation of multi-scale training. The code of multi-scale training:
`  if is_training:
    # Multi Scales
    image_size = tf.shape(reshaped_image)
    scale_value = tf.random_uniform([1],minval=256,maxval=481,dtype=tf.int32)
    scale = tf.divide(tf.cast(scale_value[0],tf.float32),tf.cast(tf.minimum(image_size[0],image_size[1]),tf.float32))
    resize_scale0 = tf.multiply(scale,tf.cast(image_size[0],tf.float32))
    resize_scale1 = tf.multiply(scale,tf.cast(image_size[1],tf.float32))
    reshaped_image = tf.image.resize_images(reshaped_image,[tf.cast(resize_scale0,tf.int32),tf.cast(resize_scale1,tf.int32)])
  else:
    #Resize image to 256*256
    reshaped_image = tf.image.resize_images(reshaped_image, args.load_size)`

**Comment by arashno** [comment]
How is the performance on the training set?
Can you share the cross-entropy loss plot?

**Comment by souxun2015** [comment]
Sure, the following two figures show these two things.
![image](https://user-images.githubusercontent.com/12790408/36879408-9b18aa8c-1dfe-11e8-9245-7b56d56cb786.png)

![image](https://user-images.githubusercontent.com/12790408/36879394-85851110-1dfe-11e8-81c8-fb8ee0ee4eed.png)

**Comment by arashno** [comment]
Interesting!
It seems that using a different learning rate and weight decay policy can help.
Have you tried the default LR and WD of the code?

**Comment by souxun2015** [comment]
I tried. But it still did not reach the accuracy. Maybe there is something we missed, I am still checking on it.

**Comment by arashno** [comment]
Is this only applies to ResNet-34?
Have you tried ResNet-18?
